wagtail/users/utils.py

```diff
@@ -25,26 +27,59 @@ def user_can_delete_user(current_user, user_to_delete):
     return True
 
 
-def get_gravatar_url(email, size=50):
-    default = "mp"
-    size = (
-        int(size) * 2
-    )  # requested at retina size by default and scaled down at point of use with css
+def get_gravatar_url(email, size=50, default_params={"d": "mp"}):
+    """
+    See https://gravatar.com/site/implement/images/ for Gravatar image options
+    Example usage:
+    ```python
+    # Basic usage
+    gravatar_url = get_gravatar_url('user@example.com')
+
+    # Customize size and default image
+    gravatar_url = get_gravatar_url(
+        'user@example.com',
+        size=100,
+        default_params={'d': 'robohash', 'f': 'y'}
+    )
+    Note: If any parameter in `default_params` also exists in the provider URL,
+    It will be overridden by the provider URL's query parameter.
+    ```
+    """
+
+    DEFAULT_GRAVATAR_PROVIDER_URL = "//www.gravatar.com/avatar"
+
+    params = {
+        **default_params,
+        # requested at retina size by default and scaled down at point of use with css
+        "s": int(size) * 2,
+    }
+
     gravatar_provider_url = getattr(
-        settings, "WAGTAIL_GRAVATAR_PROVIDER_URL", "//www.gravatar.com/avatar"
+        settings, "WAGTAIL_GRAVATAR_PROVIDER_URL", DEFAULT_GRAVATAR_PROVIDER_URL
```

Comment:
I would not worry about this change, pulling this out to a variable feels unnecessary.